test/SIL/ownership-verifier/borrow_cast_validate.sil

```diff
@@ -0,0 +1,267 @@
+
+// RUN: %target-sil-opt -sil-ownership-verifier-enable-testing -ownership-verifier-textual-error-dumper -enable-sil-verify-all=0 %s -o /dev/null 2>&1 | %FileCheck %s
```

Comment:
Nit: Can you put the RUN line on the top line and put a space in between the REQUIRES: asserts and the imports.